Header header

int16 speed_g_static
int16[13] speed_wanted
float64[13] joint_now
float64[13] joint_wanted
